data-protocols/dsp/dsp-http-spi/src/main/java/org/eclipse/edc/protocol/dsp/http/spi/dispatcher/DspHttpRemoteMessageDispatcher.java

```diff
@@ -41,10 +42,10 @@ <M extends RemoteMessage, R> void registerMessage(Class<M> clazz, DspHttpRequest
     /**
      * Registers a {@link Policy} scope to be evaluated for certain types of messages
      *
-     * @param messageClass the message type for which evaluate the policy.
-     * @param scope the scope to be used.
+     * @param <M>            the message type.
```

Comment:
this will almost certainly trip up the `javadoc` task. avoid chevrons (`"<"`) in javadoc

Reply:
looks like javadoc is not angry at it, anyway no problem in removing it